services/emailer/src/namex_emailer/email_templates/approved-ia.md

```diff
@@ -0,0 +1,69 @@
+# Results of your Name Request
+
+Your Name Request is **approved**. Follow the steps below to complete your application using this name. If the name request expires before the application is completed, a new name request will be required.
+
+---
+
+# You\'re not done yet!
+
+At the moment, there are two systems you can use to complete your application. Answer the following questions to find out which system you should use.
+
+1. Are you registering a new business in B.C. that is not already registered in other jurisdictions?
+2. Do you plan to only use the following basic filings?
+  - Incorporation Application
+  - Change of Address
+  - Change of Director
+  - Alterations
+3. Will you be the managing your own business information online?
+
+If any of your answers were NO, complete your application using,  [BC Corporate Online]({{CORP_ONLINE_URL}})
+
+If all of your answers were YES, complete your application using, [BC Registries and Online Services]({{MAGIC_LINK}})
+
+---
+
+# Steps for using BC Registries
+
+1. Go to [BC Registries and Online Services]({{MAGIC_LINK}})
+2. Login or create an account
+3. Go to My Business Registry
+4. Look up your name request
+5. Complete and submit the form along with any required documentation and payment
+
+**Connect your Name Request to your Account** 
+
+If you were not signed into your BC Registries account when you created this name request, you must connect your name request to your account by following these steps:
+
+1. Go to [BC Registries and Online Services]({{DECIDE_BUSINESS_URL}})
+2. Login or create an account
+3. Go to My Business Registry
+4. Look up your name request
+5. Enter your information and select “Manage This Name Request”
+6. The name request is successfully connected to your account if it shows on your registry table
+7. Complete and submit the form along with any required documentation and payment
```

Comment:
This is not needed at all. Confirm the steps with Jacqueline.

Unless I'm mistaken, this is all handled by the magic link.

For your information:
- magic link will ask user to login if they're not already logged in (this is why they need an account first)
- the NR will be affiliated to the current account
- if there is an existing draft IA for this NR then user is redirected to the business dashboard
- otherwise, a draft IA will be create for this NR, affiliated to the current account, and the user will be redirected to the business dashboard

Reply:
PS Same comment for the other emails with this section.

Reply:
Updated